### PR TITLE
Refactor debug log content, move data into tracef

### PR DESCRIPTION
**Pull request by antJack**
### Issues associated with this PR

Your PR should present related issues you want to solve.

### Solutions
You should show your solutions about the issues in your PR, including the overall solutions, details and the changes. At this time, Chinese is allowed to describe these.

### UT result
Unit Test is needed if the code is changed, your unit test should cover boundary cases, corner cases, and some exceptional cases. And you need to show the UT result.

### Benchmark
If your code involves the processing of every request, you should give the Benchmark Result.

### Code Style
+ Make sure `Goimports` has run
+ Show `Golint` result

### CODE REVIEW

File: pkg/proxy/downstream.go [hunk 1]
```diff
@@ -375,7 +375,8 @@ func (s *downStream) OnReceive(ctx context.Context, headers types.HeaderMap, dat
 	s.tracks = track.TrackBufferByContext(ctx).Tracks
 
 	if log.Proxy.GetLogLevel() >= log.DEBUG {
-		log.Proxy.Debugf(s.context, "[proxy] [downstream] OnReceive headers:%+v, data:%+v, trailers:%+v", headers, data, trailers)
+		log.Proxy.Debugf(s.context, "[proxy] [downstream] OnReceive")
+		log.Proxy.Tracef(s.context, "[proxy] [downstream] OnReceive headers:%+v, data:%+v, trailers:%+v", headers, data, trailers)
```
Comment (taoyuanyuan) [inline pkg/proxy/downstream.go:379]:
```suggestion
if  log.Proxy.GetLogLevel() >= log.TRACE {
		log.Proxy.Tracef(s.context, "[proxy] [downstream] OnReceive headers:%+v, data:%+v, trailers:%+v", headers, data, trailers)
```

Comment (antJack) [reply pkg/proxy/downstream.go:379]: 不需要再检查一遍了，反正都已经开 debug 了

File: pkg/proxy/downstream.go [hunk 2]
```diff
@@ -885,7 +887,8 @@ func (s *downStream) receiveData(endStream bool) {
 
 	data := s.downstreamReqDataBuf
 	if log.Proxy.GetLogLevel() >= log.DEBUG {
-		log.Proxy.Debugf(s.context, "[proxy] [downstream] receive data = %v", data)
+		log.Proxy.Debugf(s.context, "[proxy] [downstream] receive data, len = %v", data.Len())
```
Comment (taoyuanyuan) [inline pkg/proxy/downstream.go:890]: data肯定不会为nil吧？

Comment (antJack) [reply pkg/proxy/downstream.go:890]: 不会 image

**Review by taoyuanyuan — APPROVED** [review]
LGTM

**Review by doujiang24 — APPROVED** [review]

**Pull request merged**